src/core.ml

```diff
@@ -563,6 +562,9 @@ module Make (Error : ERROR) = struct
   type error = string * string list * Error.error
   type input = (string * string list) list
 
+  (* TODO [aerben] sihl sidenote: schema checked every second *)
```

Comment:
@joseferben I noticed this when debugging conformist in a Sihl app with a `print_endline`. It seems like the schema is checked every time the Sihl queue is polled, is this intended?

Reply:
I don't see why that would be the case right now if you schedule an empty job (identity function) for instance. Are you sure it's not the scheduled function doing that?

Reply:
It might be the `Sihl_email.Queued` causing this. I noted it down to investigate further at one point, this is out of scope for this issue.